docs/excel/custom-functions-dialog.md

```diff
@@ -0,0 +1,105 @@
+---
+ms.date: 03/13/2019
+description: Create dialog boxes through custom functions in Excel using JavaScript.
+title: Custom functions dialogs (preview)
+localization_priority: Priority
+---
+# Display a dialog box in custom functions
+
+Within your add-in, you have two options for creating dialogs:
```

Comment:
These bullet points confused me because my first thought was "isn't this a dialog API for custom functions?" What if we reworked as follows

If your custom function needs to interact with the user, you can create a dialog box using the [OfficeRuntime.Dialog](...linktoref...) object. A common scenario for using the dialog box is to authenticate a user so that your custom function can access a web service. For more details about authentication with custom functions, see [Custom functions authentication](...)

Note: The OfficeRuntime.Dialog object is part of the custom functions runtime. It cannot be used from the context of a task pane. To create a dialog from a task pane, see [Dialog API](...)

Reply:
Sounds good!